plugins/kotlin-dataframe/src/org/jetbrains/kotlinx/dataframe/plugin/loadInterpreter.kt

```diff
@@ -161,6 +166,11 @@ internal inline fun <reified T> String.load(): T {
         "ReadDelimStr" -> ReadDelimStr()
         "GroupByToDataFrame" -> GroupByToDataFrame()
         "ToDataFrameFrom0" -> ToDataFrameFrom()
+        "All0" -> All0()
+        "ColsOf0" -> ColsOf0()
+        "ColsOf1" -> ColsOf1()
+        "ColsAtAnyDepth0" -> ColsAtAnyDepth0()
+        "FrameCols0" -> FrameCols0()
```

Comment:
Cool list, looks like a good candidate to be a list of CONSTANTS or objects? Could you please remind? Could we reuse CONSTANTS in the @Interpretable annotations? (it makes the navigation easier)

Reply:
Hm, my concern was that they should live in a dataframe-core, but also not available in public API. But looks like it's not a problem with friend module. I'll write this down and try to refactor later